specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/managedInstanceAdministrators.json

```diff
@@ -0,0 +1,420 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-03-01-preview",
```

Comment:
>"version": "2017-03-01-preview", [](start = 4, length = 32)

I see that last three latest apis versions are in preview.. we call these perpetual preview apis :).. 
When will these be out of preview and move to stable ?

Reply:
Yes, well I am not aware of the plan or timeline when we plan to move apis to stable version and release stable version api, to be honest. Maybe @jaredmoo knows more.